definitions/consented/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1805,6 +1805,13 @@
     "UserRole": "caseworker-divorce-financialremedy-solicitor",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "CaseFieldID": "applicantAddressConfidential",
+    "UserRole": "caseworker-divorce-financialremedy-solicitor",
```

Comment:
should the solicitor have read access to this? also, I would include the authorisation for [APPSOLICITOR] like we've done for the respondent tab